flyteidl/protos/flyteidl/admin/agent.proto

```diff
@@ -157,4 +165,7 @@ message GetTaskLogsRequest {
 message GetTaskLogsResponse {
   // The execution log results.
   repeated string results = 1;
+  // In the case of multiple pages of results, the server-provided token can be used to fetch the next page
+  // in a query. If there are no more results, this value will be empty.
+  string token = 2;
```

Comment:
You'll need to add this to the request too

Reply:
I forgot it, thanks